plugins/src/lib.rs

```diff
@@ -46,9 +46,10 @@ where
     options: HashMap<String, UntypedConfigOption>,
     option_values: HashMap<String, Option<options::Value>>,
     rpcmethods: HashMap<String, RpcMethod<S>>,
+    setconfig_callback: Option<AsyncCallback<S>>,
```

Comment:
Personally i would find it tedious to write a callback for every option. I've rewritten the options to dynamic ones in one of my plugin to see how it would work and it works perfectly fine with one callback function, but again that is just me.